### DISCUSSION TITLE
5. November 2024

### DISCUSSION THREAD

**Discussion by mkow** [gramineproject/gramine#2049] [Meeting notes]
Agenda:

**Comment by kailun-qin** [comment]
Kailun: discuss encrypted files recovery feature support in Gramine (option: porting Intel SGX SDK's design, see also https://github.com/gramineproject/gramine/issues/2013 and https://github.com/gramineproject/gramine/discussions/1972).

**Comment by donporter** [comment]
Don: Short discussion: Gramine was selected as finalist for the ACSAC Impact prize.  Someone needs to give a 10 min presentation; input on revisions to submission document also appreciated.
